src/SDKs/Graph.RBAC/Graph.RBAC/Generated/GroupsOperationsExtensions.cs

```diff
@@ -132,37 +136,6 @@ public static async Task AddMemberAsync(this IGroupsOperations operations, strin
                 (await operations.AddMemberWithHttpMessagesAsync(groupObjectId, parameters, null, cancellationToken).ConfigureAwait(false)).Dispose();
             }
 
-            /// <summary>
-            /// Delete a group from the directory.
-            /// </summary>
-            /// <param name='operations'>
-            /// The operations group for this extension method.
-            /// </param>
-            /// <param name='groupObjectId'>
-            /// The object ID of the group to delete.
-            /// </param>
-            public static void Delete(this IGroupsOperations operations, string groupObjectId)
```

Comment:
Looks like there are some breaking changes here, please bump the Assembly version in csproj and AssemblyInfo.cs files to 3.5.0-preview

Reply:
Where is the .txt log file? Updated the version numbers.